packages/aragon-rpc-messenger/src/signals.js

```diff
@@ -0,0 +1,5 @@
+export const COMPLETE = Symbol('RPC_COMPLETE')
```

Comment:
Could this pose problems given that symbols can't be serialised for `postMessage`?
https://developer.mozilla.org/en-US/docs/Web/API/Web_Workers_API/Structured_clone_algorithm

Reply:
Ah yes, this was designed around that; the receiver actually imports this symbol from `@aragon/rpc-messenger` and uses it directly within their context, so it never gets sent through a `postMessage`.